lib/created_message_queue_handler.ts

```diff
@@ -276,6 +311,21 @@ export async function handleMessage(
   //  Email notification
   //
 
+  // check if the user has blocked emails sent from this service
+  // 'some(true)' in case we must send the notification by email
+  // 'none' in case the user has blocked the email channel
+  const maybeIsUnblockedEmail = fromPredicate(
```

Comment:
negative booleans are hard to keep track of (mentally): `maybeIsUnblockedEmail` -> `maybeIsEmailAllowed`